### DISCUSSION TITLE
how to fix?

### DISCUSSION THREAD

**Discussion by alexandrucn** [FunkinCrew/Funkin#2061] [Compiling help]
how to fix this error? Video : https://www.youtube.com/watch?v=6ldO2W7HNUM

C:/HaxeToolkit/haxe/lib/polymod/git/polymod/hscript/_internal/PolymodScriptClass.hx:60: characters 52-56 : polymod.hscript._internal.ErrorEx has no field line
C:/HaxeToolkit/haxe/lib/polymod/git/polymod/hscript/_internal/PolymodScriptClass.hx:63: characters 87-91 : polymod.hscript._internal.ErrorEx has no field line
source/NGio.hx:12: characters 1-68 : io.newgrounds.objects.events.Result has no field or subtype GetCurrentVersionResult
source/NGio.hx:13: characters 1-61 : io.newgrounds.objects.events.Result has no field or subtype GetVersionResult
source/NGio.hx:47: characters 62-76 : io.newgrounds.Call<io.newgrounds.objects.events.GetCurrentVersionData> has no field addDataHandler (Suggestion: addStatusHandler)
source/NGio.hx:47: characters 104-127 : Type not found : GetCurrentVersionResult
source/NGio.hx:97: characters 25-39 : Void -> Void should be Null<io.newgrounds.objects.events.Outcome<io.newgrounds.CallError> -> Void>
source/NGio.hx:97: characters 25-39 : For optional function argument 'callback'
source/NGio.hx:100: characters 30-45 : Void -> Void should be Null<io.newgrounds.objects.events.Outcome<io.newgrounds.CallError> -> Void>
source/NGio.hx:100: characters 30-45 : For optional function argument 'callback'
source/NGio.hx:86: characters 25-34 : Void -> Void should be Null<io.newgrounds.LoginOutcome -> Void>
source/NGio.hx:86: characters 25-34 : For optional function argument 'callback'
C:/HaxeToolkit/haxe/lib/polymod/git/polymod/hscript/_internal/PolymodScriptClass.hx:366: characters 69-73 : polymod.hscript._internal.ErrorEx has no field line
C:/HaxeToolkit/haxe/lib/polymod/git/polymod/hscript/_internal/PolymodInterpEx.hx:73: characters 14-43 : polymod.hscript._internal.ErrorEx should be hscript.Error
C:/HaxeToolkit/haxe/lib/polymod/git/polymod/hscript/_internal/PolymodInterpEx.hx:73: characters 14-43 : For function argument 'e'
C:/HaxeToolkit/haxe/lib/polymod/git/polymod/hscript/_internal/PolymodInterpEx.hx:450: characters 13-38 : polymod.hscript._internal.ErrorEx should be hscript.Error
C:/HaxeToolkit/haxe/lib/polymod/git/polymod/hscript/_internal/PolymodInterpEx.hx:450: characters 13-38 : For function argument 'e'
C:/HaxeToolkit/haxe/lib/polymod/git/polymod/hscript/_internal/PolymodInterpEx.hx:105: characters 13-40 : polymod.hscript._internal.ErrorEx should be hscript.Error
C:/HaxeToolkit/haxe/lib/polymod/git/polymod/hscript/_internal/PolymodInterpEx.hx:105: characters 13-40 : For function argument 'e'
C:/HaxeToolkit/haxe/lib/polymod/git/polymod/hscript/_internal/PolymodInterpEx.hx:483: characters 13-38 : polymod.hscript._internal.ErrorEx should be hscript.Error
C:/HaxeToolkit/haxe/lib/polymod/git/polymod/hscript/_internal/PolymodInterpEx.hx:483: characters 13-38 : For function argument 'e'

**Comment by Snirozu** [comment]
check [#2055](https://github.com/ninjamuffin99/Funkin/issues/2055#issuecomment-1287238664)

**Reply by alexandrucn** [reply]
thanks it work
